src/components/CRow.vue

```diff
@@ -1,17 +1,46 @@
 <template>
   <tr class="border-b border-gray-200 hover:bg-blue-100">
+    <td class="border px-4 py-2">
+      <pre class="text-xs whitespace-normal">{{ priority }}</pre>
+    </td>
     <td class="border px-4 py-2">
       <pre class="text-xs whitespace-normal">{{ message_id }}</pre>
     </td>
     <td class="border px-4 py-2 font-semibold text-sm " :style="{ color: getColorState() }">
-      <pre class="text-xs whitespace-normal">{{ name }}</pre>
+      <pre class="text-xs whitespace-normal">{{ nameState }}</pre>
     </td>
     <td class="border px-4 py-2">
       <pre class="text-xs whitespace-normal">{{ args }}</pre>
     </td>
     <td class="border px-4 py-2">
       <pre class="text-xs whitespace-normal">{{ kwargs }}</pre>
     </td>
+    <td class="border px-4 py-2" :class="{ 'bg-green-200': nameState === 'Pending' }">
```

Comment:
There is way too many columns ... we should only keep 3: 
- started datetime
- wait time
- execution time